securedrop_client/logic.py

```diff
@@ -406,6 +406,7 @@ def __init__(  # type: ignore [no-untyped-def]
         # Create a timer to show the time since the last sync
         self.show_last_sync_timer = QTimer()
         self.show_last_sync_timer.timeout.connect(self.show_last_sync)
+        self.show_last_sync_timer.start(TIME_BETWEEN_SHOWING_LAST_SYNC_MS)
```

Comment:
This is the time that passed before the "last sync" widget is refreshed, am I right?

What would you think of renaming the constant something along the lines of: `LAST_SYNC_REFRESH_INTERVAL_MS`?

Reply:
Yep that's right. That naming makes sense to me 👍